src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -3180,7 +3215,6 @@ static void updateEntity
   std::string     locAttr = "";
   BSONObj         currentGeoJson;
   BSONObjBuilder  geoJson;
-
```

Comment:
Again, the empty lines (well, most of them anyway), are there for a reason.

Reply:
Fixed in abdaa28